src/components/MenuButton/MenuButton.module.scss

```diff
@@ -20,8 +20,11 @@
   background-color: var(--primary-background-hover-color);
 }
 
-.active {
+button:active {
```

Comment:
Wasn't the task to remove the scaling when the button is selected but not active? It looks like it's still scaling down

Reply:
It is supposed to scale while the user clicks (before he let go of the mouse)